python/ray/scripts/scripts.py

```diff
@@ -601,6 +601,13 @@ def start(
             stacklevel=2,
         )
 
+    if os.getenv("ENABLE_TEE") == "true":
```

Comment:
In PR #380 , I used environment variable `RAY_IN_TEE` ,  let's choose a unified name

Reply:
Yes, I noticed that. I'm not sure what the best name is here. Maybe let's @ others CC